src/api/routes/signup.js

```diff
@@ -1,8 +0,0 @@
-import router from 'express';
```

Comment:
@murediane why did you delete this file?

Reply:
It was the route to signup that was just used to validate data and it is no longer needed since the middleware is called in the signup route. It was just useless